network/config.go

```diff
@@ -38,9 +39,11 @@ var DefaultConfig = Config{
 	discovery:  time.Second * 2,
 }
 
+var defaultLogger, _ = zap.NewDevelopment()
```

Comment:
Should still be a public variable, so people are free to use it explicitly if they want.